ConwaysGameOfLife.Tests/ValidatorShould.cs

```diff
@@ -55,7 +56,7 @@ public void AcceptLocation_WhenInputIsValid(string input, int gridWidth, int gri
         [InlineData("a,b", 3, 3)]
         [InlineData("-1,4", 3, 3)]
         [InlineData("4,-1", 3, 3)]

```

Comment:
Same here, can we get some different size grid widths and lengths? What happens if we pass in negative value widths and lengths? (Not that we would)